src/MvcMusicStore.Spa/MvcMusicStore.Spa.csproj

```diff
@@ -18,9 +18,9 @@
     <MvcBuildViews>false</MvcBuildViews>
     <UseIISExpress>true</UseIISExpress>
     <IISExpressSSLPort />
-    <IISExpressAnonymousAuthentication />
-    <IISExpressWindowsAuthentication />
-    <IISExpressUseClassicPipelineMode />
+    <IISExpressAnonymousAuthentication>disabled</IISExpressAnonymousAuthentication>
```

Comment:
They were automatic. I have removed them from the PR I will make to Dev. As I said, I know this is a minor point and I appreciate your in depth talk. I know technically that it is an method with accessors, but for readability, I put basic methods with accessor below fields to enhance readability of a class. Regardless of the outcome of the PR, I'm a super fan of you @Eilon and @Praburaj